seed/src/lib.rs

```diff
@@ -317,3 +365,28 @@ impl Node {
         Ok(())
     }
 }
+
+/// Guess a user given a peer id.
+async fn guess_user(
+    peer: PeerId,
+    api: &PeerApi<Signer>,
+) -> Result<Option<User<entity::Draft>>, Error> {
+    api.with_storage(move |s| {
+        let metas = s.all_metadata()?;
+
+        for meta in metas {
+            let meta = meta?;
+            let repo = s.open_repo(meta.urn())?;
+
+            for remote in repo.tracked()? {
```

Comment:
You could just do `s.tracked(meta.urn())?`

Reply:
Hmm the thing is I use `repo` right below..

Reply:
```rust
                    let user = repo.get_rad_self_of(remote)?;
```

Reply:
Oh aye, I mean you could do `s.get_rad_self_of(meta.urn(), remote)` but I'm splitting hairs at that stage. Disregard :)